music21/expressions.py

```diff
@@ -325,6 +325,114 @@ def nextMark(self):
         return RehearsalMark(self.nextContent(), numbering=self.numbering)
 
 
+# ------------------------------------------------------------------------------
+class PedalMark(spanner.Spanner):
+    '''
+        A pedal mark spanner contains a pedaled set of notes.
+        The spanner starts at the moment of down-pedal, and
+        ends at the moment of up-pedal.  There can be any
+        number of pedal "bounces" in the middle of the spanner.
+
+        The visual form of the pedal mark can vary.  The following
+        examples use a pedal mark with one "bounce" in the middle.
+
+        Pedal marks can be lines:            |_______^________|
+        Pedal marks can be normal symbolic:  Ped.    * Ped.   *
+        Pedal marks can be altered symbolic: Ped.    Ped.     *
+        Pedal marks can be symbol and line:  Ped.____^________|
```

Comment:
there's a way of marking this in RST so it will appear in monospaced font in the docs, otherwise this will be gibberish on the web.